Formula/pyside.rb

```diff
@@ -19,46 +19,60 @@ class Pyside < Formula
 
   depends_on "cmake" => :build
   depends_on "ninja" => :build
+  depends_on xcode: :build
   depends_on "llvm"
   depends_on "python@3.9"
   depends_on "qt"
 
+  uses_from_macos "libxml2"
+  uses_from_macos "libxslt"
+
   def install
-    xy = Language::Python.major_minor_version Formula["python@3.9"].opt_bin/"python3"
+    inreplace "sources/pyside6/cmake/Macros/PySideModules.cmake",
+              "${shiboken_include_dirs}",
+              "${shiboken_include_dirs}:#{Formula["qt"].opt_include}"
 
-    args = std_cmake_args + %W[
-      -GNinja
-      -DPYTHON_EXECUTABLE=#{Formula["python@3.9"].opt_bin}/python#{xy}
-      -DCMAKE_INSTALL_RPATH=#{lib}
+    qt = Formula["qt"]
+    site_packages = prefix/Language::Python.site_packages("python3")
+    site_pyside = site_packages/"PySide6"
+    pyside_args = %w[
+      --no-examples
+      --no-qt-tools
+      --rpath @loader_path/../shiboken6
+      --shorter-paths
+      --skip-docs
     ]
+    system "python3", *Language::Python.setup_install_args(prefix), *pyside_args
 
-    mkdir "build" do
-      system "cmake", *args, ".."
-      system "ninja", "install"
-    end
+    # install tools symlinks
+    %w[lupdate lrelease].each { |x| site_pyside.install_symlink qt.opt_bin/x }
+    %w[uic rcc].each { |x| (site_pyside/"Qt/libexec").install_symlink qt.opt_pkgshare/"libexec"/x }
+    %w[assistant linguist].each { |x| site_pyside.install_symlink qt.opt_bin/x.capitalize => x }
+    site_pyside.install_symlink qt.opt_libexec/"Designer.app"
```

Comment:
`install_symlink` to a file in another keg will create a link that references a Cellar path. We don't want that because it breaks on upgrades.

Use `ln_s` instead. See `julia`, `include-what-you-use`, or any formulae that depends on `macos-term-size` for examples.